app/controllers/application_controller.rb

```diff
@@ -1,2 +1,35 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :null_session
```

Comment:
I had some struggles with getting `Rack::Cors` to behave as expected, so I used `protect_from_forgery` with an empty session to handle unverified requests in the meantime. This is a piece of technical debt to clean up.